packages/ui/src/tabs/TabsOverrides.ts

```diff
@@ -7,6 +7,7 @@ export function overrideTabs(theme: SuperDispatchTheme): void {
     textColor: 'primary',
     indicatorColor: 'primary',
   };
+  theme.palette.primary.main = ColorDynamic.Blue500;
```

Comment:
Lets move it to ThemeProvider

Reply:
cant, because there is already primary.main set to Blue300, so i needed to change it for only this file